.buildkite/scripts/stack_tests_windows.ps1

```diff
@@ -30,7 +30,7 @@ function withDocker($version) {
 
```

Comment:
Current version defined in our pipeline for docker (26.1.2) is not available in chocolatey: https://community.chocolatey.org/packages/docker-engine#versionhistory
So not too much we can do there.

Probably it could be added also the flag `--allow-downgrade` in case it exists the next docker version we set eventually. Or just keep it as it is now.

Reply:
> Current version defined in our pipeline for docker (26.1.2) is not available in chocolatey: https://community.chocolatey.org/packages/docker-engine#versionhistory
> So not too much we can do there.

Is this producing any problem? Could we change to 26.1.4, that is available in chocolatey?